icontrol/session.py

```diff
@@ -292,8 +311,11 @@ def __init__(self, username, password, **kwargs):
             self.session.auth = (username, password)
 
         # Set state as indicated by ancestral code.
-        self.session.verify = False  # XXXmake TOFU
-        self.session.headers.update({'Content-Type': 'application/json'})
+        self.session.verify = False  # XXX make TOFU
+        self.session.headers.update({
```

Comment:
@pjbreaux. Yes. There are no existing functional tests that validate a user agent